packages/nimble-components/src/label-provider/core/index.ts

```diff
@@ -10,7 +10,9 @@ import {
     popupIconInformationLabel,
     filterSearchLabel,
     filterNoResultsLabel,
-    loadingLabel
+    loadingLabel,
+    scrollBackwardLabel,
```

Comment:
The quarterly maintenance task isn't something to rely on (it's a backstop to forgetting), should expect to do it in a follow-up PR soon so that the new needed localized strings propagate around sooner.